modules/form-field/react/spec/FormField.spec.tsx

```diff
@@ -1,152 +1,109 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
-import {GrowthBehavior} from '@workday/canvas-kit-react-common';
-import FormField, {FormFieldErrorBehavior} from '../lib/FormField';
+import {render} from '@testing-library/react';
+import FormField from '../lib/FormField';
+import {ErrorType} from '@workday/canvas-kit-react-common';
 
 describe('FormField', () => {
-  test('Set a string label', () => {
-    const label = 'Label';
-    const component = mount(
-      <FormField label={label}>
-        <input type="text" />
-      </FormField>
-    );
-
-    expect(component.find('label').text()).toBe(label);
-
-    component.unmount();
-  });
-
-  test('Set a custom component', () => {
-    const label = <label>Custom Label</label>;
-    const component = mount(
-      <FormField label={label}>
-        <input type="text" />
-      </FormField>
-    );
-
-    expect(component.contains(label)).toBeDefined();
-
-    component.unmount();
-  });
-
-  test('Set hint text', () => {
-    const hintText = 'Hint Text';
-    const component = mount(
-      <FormField hintText={hintText}>
-        <input type="text" />
-      </FormField>
-    );
-
-    expect(component.render().text()).toEqual(expect.stringContaining(hintText));
-
-    component.unmount();
+  const cb = jest.fn();
+  afterEach(() => {
+    cb.mockReset();
   });
 
-  test('Set aria-describedby', () => {
-    const InputComponent: React.FunctionComponent<GrowthBehavior> = () => <input type="text" />;
-    const hintText = 'Hint Text';
-    const hintId = 'hint-id';
-    const component = mount(
-      <FormField error={FormField.ErrorType.Error} hintText={hintText} hintId={hintId}>
-        <InputComponent />
-      </FormField>
-    );
-
-    expect(
-      component
-        .find('Hint')
-        .at(0)
-        .prop('id')
-    ).toEqual(hintId);
-    expect(component.find(InputComponent).prop('aria-describedby')).toEqual(hintId);
-
-    component.unmount();
+  describe('when rendered', () => {
+    it('should render a label with text Label', () => {
+      const label = 'Label';
+      const {container} = render(
```

Comment:
`getByLabelText` is much cleaner. `container` should be a last resort

Reply:
In this case, I'm looking for the `Label` and the text associated with that label, `getByLabelText` will return any element associated with that label, which is why i used `container` instead. I have a talked with @NicholasBoll about this and how it was confusing to me that hat method really grabs elements associated with it and not the node `Label`

Reply:
Whoops - good call out haha. I think I was thinking of `getByText`.